ci/symfony.yml

```diff
@@ -0,0 +1,28 @@
+name: Symfony
+
+on:
+  push:
+    branches: [ $default-branch ]
+  pull_request:
+    branches: [ $default-branch ]
+
+jobs:
+  symfony-tests:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: shivammathur/setup-php@15c43e89cdef867065b0213be354c2841860869e
+      with:
+        php-version: '8.0'
+    - uses: actions/checkout@v2
+    - name: Copy .env.test.local
+      run: php -r "file_exists('.env.test.local') || copy('.env.test', '.env.test.local');"
+    - name: Install Dependencies
```

Comment:
The default [php](https://github.com/actions/starter-workflows/blob/main/ci/php.yml) template uses composer cache - https://github.com/actions/starter-workflows/blob/7b64f441651d3005d0849daa72311c22052441d6/ci/php.yml#L20
Can we use it here as well?